src/electron/lib/api/apis/GraphApi.ts

```diff
@@ -9,20 +9,9 @@ import { type NodeSignature } from "@shared/ui/ToolboxTypes";
 // Graphs across projects are stored homogeneously and referenced by UUID
 export class GraphApi implements ElectronMainApi<GraphApi> {
   private readonly _blix: Blix;
-  private readonly graphSubscriber: IPCGraphSubscriber;
```

Comment:
See [here](https://github.com/COS301-SE-2023/AI-Photo-Editor/pull/82/files#r1267763845) why I removed this.